lua/entities/gmod_wire_expression2/core/vehicle.lua

```diff
@@ -0,0 +1,146 @@
+--print("Loading Vehicle-Entity support E2 extension...")
+
+if not E2Lib.GetExtensionStatus("vehicle") then
+	hook.Remove("PlayerLeaveVehicle", "E2SetVehicleExitPoint")
+	hook.Remove("EntityRemoved", "E2RemoveVehicleExitPoint")
+end
+
+E2Lib.RegisterExtension("vehicle", true, "todo description.", "todo warning.")
+
+--------------------------------------------------------------------------------
+--  Vehicle-Entity support
+--------------------------------------------------------------------------------
+
+local IsValid = IsValid
+local isOwner = E2Lib.isOwner
+
+-- Localization; Faster access to bit/math/util library functions
+local bit = bit
+local bit_band = bit.band
+local math = math
+local math_max = math.max
+local math_min = math.min
+local util = util
+--local util_TraceHull = util.TraceHull
+local util_PointContents = util.PointContents
+
+registerCallback("e2lib_replace_function", function(funcname, func, oldfunc)
+	if funcname == "IsValid" then
+		IsValid = func
+	elseif funcname == "isOwner" then
+		isOwner = func
+	end
+end)
```

Comment:
I think you should just call E2Lib.isOwner directly without bothering to localize it, and I don't think IsValid is ever replaced - its a Garry global I believe.

In general localization like this is very much a micro-optimization anyway.